src/main/java/io/snyk/jenkins/SnykStepBuilder.java

```diff
@@ -210,20 +209,6 @@ public boolean perform(@Nonnull AbstractBuild<?, ?> build, @Nonnull Launcher lau
         throw new AbortException("Snyk API token with ID '" + snykTokenId + "' was not found. Please configure the build properly and retry.");
       }
       envVars.put("SNYK_TOKEN", snykApiToken.getToken().getPlainText());
-      envVars.overrideAll(build.getBuildVariables());
```

Comment:
I made a parameterised build and confirmed that the build parameters are available in `envVars` without having to do this step.